eng/pipelines/branch-cleanup.yml

```diff
@@ -1,17 +1,44 @@
 parameters:
-- name: Repos
-  type: object
-  default:
-    - azure-sdk-for-android
-    - azure-sdk-for-c
-    - azure-sdk-for-cpp
-    - azure-sdk-for-go
-    - azure-sdk-for-ios
-    - azure-sdk-for-java
-    - azure-sdk-for-js
-    - azure-sdk-for-net
-    - azure-sdk-for-python
-
+Repos:
```

Comment:
Given this is an object structure I wonder if we should group things together so we can share some of the settings like the pattern. Also just to make things a little more compact should we put the repo name like "Azure/azure-sdk-for-android"?